myosuite/envs/myo/reorient_sar_v0.py

```diff
@@ -0,0 +1,406 @@
+""" =================================================
+# Copyright (c) Facebook, Inc. and its affiliates
+Authors  :: Cameron Berg (cameronberg@meta.com), Vikash Kumar (vikashplus@gmail.com), Vittorio Caggiano (caggiano@gmail.com)
+================================================= """
+
+import collections
+import numpy as np
+import gym
+
+from myosuite.envs.myo.base_v0 import BaseV0
+from myosuite.utils.quat_math import euler2quat, mulQuat, negQuat, mat2quat
+from myosuite.utils.vector_math import calculate_cosine
+
+class ProprioceptiveEnvV0(BaseV0):
+
+    DEFAULT_OBS_KEYS =['hand_jnt','obj_pos','obj_vel','obj_rot','obj_des_rot','obj_err_pos','obj_err_rot','act','mlen','mvel','mforce']
+    
+    DEFAULT_RWD_KEYS_AND_WEIGHTS= {
+        'pos_align':1.0,
+        'rot_align':1.0,
+        'act_reg':5.,
+        'drop':5.0,
+        'bonus':10.0
+    }
+
+    def __init__(self, model_path, obsd_model_path=None, seed=None, **kwargs):
```

Comment:
We had a requirement that all init should be at the leaf env. [details here](https://github.com/cameronberg/myosuite/blob/dev/myosuite/envs/myo/key_turn_v0.py#L27-L38)

If you have managed to be around this, I'd love to learn more. Be very careful, otherwise.

Reply:
my mistake, it is the identical process I just deleted the commented explanation, just added it back in newest commit

Reply:
I don't see it in most of the envs here 
- https://github.com/cameronberg/myosuite/blob/dev/myosuite/envs/myo/reorient_sar_v0.py#L136
- https://github.com/cameronberg/myosuite/blob/dev/myosuite/envs/myo/reorient_sar_v0.py#L209
- https://github.com/cameronberg/myosuite/blob/dev/myosuite/envs/myo/reorient_sar_v0.py#L279
- https://github.com/cameronberg/myosuite/blob/dev/myosuite/envs/myo/reorient_sar_v0.py#L349

Reply:
That is because these are all inherited from the `ProprioceptiveEnvV0` class where this has already been updated. All of the envs you linked inherit the same init where the commented explanation is included

Reply:
What is the recommended change? I can confirm these environments all work correctly in their current state—this is also confirmed by test_myo.py